cmd/influxd/restore/restore.go

```diff
@@ -156,13 +159,21 @@ func (cmd *Command) unpackMeta() error {
 	select {
 	case err := <-store.Err():
 		return err
+	default:
 	}
 
+	client := meta.NewClient([]string{store.HTTPAddr()}, false)
+	client.SetLogger(log.New(ioutil.Discard, "", 0))
```

Comment:
I'm not sure we should disable logging here.  If it fails to connect, it could be hard to tell what actually went wrong.  Thoughts?